include/geos/algorithm/distance/Makefile.am

```diff
@@ -1,9 +1,9 @@
 #
-# This file is part of project GEOS (http://trac.osgeo.org/geos/)
+# This file is part of project GEOS (http://trac.osgeo.org/geos/) 
 #
-SUBDIRS =
+SUBDIRS = 
 
-EXTRA_DIST =
+EXTRA_DIST = 
```

Comment:
> These are still unrelated to the PR
just restored the original trailing spaces to remove unrelated changes to the PR.